src/Microsoft.DotNet.Wpf/src/DirectWriteForwarder/DirectWriteForwarder.vcxproj

```diff
@@ -126,4 +129,11 @@
       <SourceRevisionId Condition="'$(SourceRevisionId)' == ''">unknownrev</SourceRevisionId>
     </PropertyGroup>
   </Target>
+  <Target Condition="'$(TargetFrameworkIdentifier)'=='.NETFramework'" Name="FixAssemblyReferences" BeforeTargets="ResolveAssemblyReferences">
+    <ItemGroup>
+      <Reference Remove="$(FrameworkPathOverride)\WindowsBase.dll" />
+      <Reference Remove="$(PkgSystem_IO_Packaging)\lib\net462\System.IO.Packaging.dll" />
+      <Reference Include="$(PkgSystem_IO_Packaging)\lib\netstandard2.0\System.IO.Packaging.dll" />
```

Comment:
When you set the correct AssetTargetFallback above, these can be removed.

Reply:
No they can't, this has nothing to do with a missing package and has to do with the fact we can't reference the old 4.0 WindowsBase, and we can't reference the NET or NETFX versions of System.IO.Packaging and need the netstandard one instead.

Reply:
System.IO.Packaging doesn't exist as a separate assembly in .NETFramework.  It's home is WindowsBase  you should be able to just condition out the SIOP reference for NETFramework and reference WindowsBase.  If that causes any project cycles then we can add the API to the cycle-breaker references under NETFramework IFDEF.